.travis.yml

```diff
@@ -3,14 +3,18 @@ language: node_js
 node_js:
   # we recommend testing addons with the same minimum supported node version as Ember CLI
   # so that your addon works for all apps
-  - "4"
+  - "6"
 
-sudo: false
+sudo: required
 dist: trusty
 
 addons:
   chrome: stable
 
+before_script:
+  - "sudo chown root /opt/google/chrome/chrome-sandbox"
```

Comment:
what do these do?

Reply:
@skaterdav85 https://github.com/travis-ci/travis-ci/issues/9024 ^^

Reply:
If you remove those, does it fail? I'm assuming other addons are fine without it since the default `.travis.yml` doesn't have it.

Reply:
Yes, without these lines the build fails, you can see the errors here: 
https://travis-ci.org/skaterdav85/ember-cli-doorkeeper/builds/355545786?utm_source=github_status&utm_medium=notification

Which led to finding https://github.com/travis-ci/travis-ci/issues/9024